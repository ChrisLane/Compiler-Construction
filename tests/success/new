// This tests the 'new' expression. This should parse.
main() {
    int x = 1;
}
