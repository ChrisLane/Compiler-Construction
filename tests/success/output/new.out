main (  ) { 
    New ( "x" = Const 1 ) In { 
        empty 
    } 
}
